apps/lumberjack-app/src/app/app.component.ts

```diff
@@ -1,10 +1,28 @@
-import { Component } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
+
+import { AppLogger } from './app.logger';
+import { ForestService } from './forest.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss'],
 })
-export class AppComponent {
+export class AppComponent implements OnInit, OnDestroy {
+  #subscriptions = new Subscription();
+
   title = 'lumberjack-app';
+
+  constructor(private logger: AppLogger, private forest: ForestService) {}
+
+  ngOnInit(): void {
+    this.logger.helloForest();
+
+    this.#subscriptions.add(this.forest.fire$.subscribe(() => this.logger.forestOnFire()));
+  }
+
+  ngOnDestroy(): void {
+    this.#subscriptions.unsubscribe();
+  }
 }
```

Comment:
Should I close my open PR? If so, we need to bring the `package.json` for the lumberjack-app.

Reply:
I created this based on our README file. I don't remember what's in `lumberjack-app` in the main repo but let's have a look after this PR has been merged.

Reply:
About that `package.json`, I'm starting to be confused about why anyone would want `"sideEffects": false` in an application. The purpose of an application is to have side effects. For us in particular, it doesn't matter at all since this is not a real-world app, just one used for testing and demonstration purposes.

Reply:
That PR will be probably closed after this one because it doesn't add anything valuable.